processes/org-monthly-meetings/minutes/2019-06-Minutes.md

```diff
@@ -0,0 +1,173 @@
+
+**Owner:** @daverick
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## On-boarding process document
+The on-boarding document has been created and has been merged. We also tested the document with Raquel and Salva during an on-boarding session. If the document needs to be updated, members can create a PR to update.
+
+**Owner:** @amaliacardenas
+
+**Status:**
+This initiative can be **closed**
+
+
+## E-mail templates
+Amalia will follow up with the London team to see if the Barcelona team can have codebar emails.
+
+As a group we identified that it can be confusing to know who i should answer an email if we get contacted by somebody. In order to make things more clear, we decided to add a "on call" field on the trello card for each workshop that we are organizing. The person who is on call for that week, is responsible for answering codebar emails.    
+
+**Owner:** @amaliacardenas
+
+**Status:**
+This initiative can be **closed**
+
+
+## Handbook for organisers
+There are no updates for this initiative. Amalia is still working on it.   
+
+**Owner:** @amaliacardenas
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## Raquel and Salva on-boarding
+
+**Owner:** @jmbriano
+Martin, Amalia  Raquel and Salva met for an on-boarding session. We are working on making the process of on-boarding new organizers smoother. We covered the main sections for organizing a workshop. There is documentation in place for other processes. Salva and Raquel will let us know if anything in the documentation needs to be updated or improved.  
+**Status:**
+This initiative can be **closed**
+
+## Instagram account
+The Instagram account is open now. Raquel has the passwords. Amalia will create a last Pass account to manage the passwords and will share it with the group. Raquel has a lot of ideas of ways to improve the Instagram account and will add a readme file with the hash tags, and how instructions on how to upload content and when to upload content so that other organizers can help manage the Instagram account.  
+
+**Owner:** @raquelalvear
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+## Cloning RGSoC
+We have decided to pilot a program called codebar teams, and not organize a RGSoC clone.  
+
+**Owner:** @loriking
+
+**Status:**
+This initiative can be **closed**
+
+## Hosting events at King
+At the moment we only have an opening in October. Once we start planning for 2020 we will contact the King team to book more dates. We will work with the team at King to simplify the organization of workshops so that we can have the event happen at king more regularly.
+
+**Owner:** @amaliacardenas
+
+**Status:**
+This initiative can be **closed**
+
+# Adding a community manager
+Lori will ask around to see if there is somebody interested in taking on this role.  
+
```

Comment:
Can someone share the objective behind this initiative? is it to get more students? more coaches? a better image?

Reply:
One of the reasons this popped up was because the "on call" for email responses, to see if it would be interesting to have a dedicated person for this, and the idea of community manager came up, as a person who could have more responsability, but also provide some additional value, such as the ones you're mentioning @jmbriano . 

It was a bit of a brainstorming, and just saying that we can explore it and see if it is worth having or not :)